### PR TITLE
fix: fix embedded url misalignment

### PR DESCRIPTION
**Pull request by Irongade**
This PR fixes some alignment issues. 

[LOOM DEMO](https://www.loom.com/share/dc2c832c8ebb41e5966ff8f99cc1ad98)

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: fix/fix-embedded-url-misallignment 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 55.83 **(0.01)** | 37.16 **(0)** | 35.21 **(0)** | 56.14 **(0)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.94 **(0.23)** | 41.67 **(0.84)** | 36.21 **(0)** | 56.99 **(0.25)**</details>

### CODE REVIEW

**Review by danieldare — APPROVED** [review]

File: app/client/src/components/editorComponents/form/fields/EmbeddedDatasourcePathField.tsx [hunk 1]
```diff
@@ -94,6 +94,7 @@ const DatasourceContainer = styled.div`
     }
     .CodeEditorTarget {
       z-index: ${Indices.Layer5};
+      margin-top: 1.8px;
```
Comment (mohanarpit) [inline app/client/src/components/editorComponents/form/fields/EmbeddedDatasourcePathField.tsx:97]: QQ: Is custom CSS required here? Is this component not a part of our design system?

File: app/client/src/components/editorComponents/EditableText.tsx [hunk 2]
```diff
@@ -78,6 +80,10 @@ const TextContainer = styled.div<{ isValid: boolean; minimal: boolean }>`
       }
     }
   }
+
+  & span.bp3-editable-text-content {
```
Comment (mohanarpit) [inline app/client/src/components/editorComponents/EditableText.tsx:84]: Same here. Is this a design system change? Or is this component not a part of the design system?

Comment (Irongade) [reply app/client/src/components/editorComponents/EditableText.tsx:84]: This is just the CSS class name for the design component we are using.

File: app/client/src/components/editorComponents/form/fields/EmbeddedDatasourcePathField.tsx (hunk 1 above)
Comment (Irongade) [reply app/client/src/components/editorComponents/form/fields/EmbeddedDatasourcePathField.tsx]: Yes, sometimes styles of the Parent elements or wrappers of design system component can affect the outlook of the component itself, causing misalignment issues, so sometimes custom css is needed.

**Pull request merged**